extend_ddl_commands.c

```diff
@@ -641,7 +641,7 @@ DeparseCreateStmt(CreateStmt *createStmt, Oid masterRelationId)
 							errdetail("Only the CHECK form is supported.")));
 		}
 
-		if (firstConstraintPrinted)
+		if (firstAttributePrinted || firstConstraintPrinted)
```

Comment:
At first I thought maybe this was unnecessary, and perhaps the code could be simplified by always printing a comma before each constraint. After all, shouldn't we have printed a column by the time we get here?

Then I remembered I frequently declare zero-column tables in testing, and wondered whether I could have a table with zero columns but still some constraints:

```sql
CREATE TABLE empty_with_constraint (CHECK (1 > 2));
-- Table "public.empty_with_constraint"
-- ┌────────┬──────┬───────────┐
-- │ Column │ Type │ Modifiers │
-- ├────────┼──────┼───────────┤
-- └────────┴──────┴───────────┘
-- Check constraints:
--     "empty_with_constraint_check" CHECK (1 > 2)
```

This is of course insane, but it shows that the check you've written is necessary.